.flake8

```diff
@@ -2,7 +2,7 @@
 # Keep in sync with setup.cfg which is used for source packages.
 
 [flake8]
-ignore = E203, E302, E501, E999
+ignore = E203, E302, E501, E999, W503
```

Comment:
Following [`black`'s recommendation](https://black.readthedocs.io/en/stable/guides/using_black_with_other_tools.html?highlight=W503#flake8) - W503 is actually disabled by default, but enabled for bugbear by the `W` in `select = B,C,E,F,W,B9`.  I updated the config because I *also* added the first case of this pattern, [in `check_for_b018()` here](https://github.com/PyCQA/flake8-bugbear/pull/181/files#diff-21e991c5e10f0aca96647b43b4d0968e75c52cc2bceaacf0794f50ded872e49bR461-R465).

Reply:
Maybe use `extend-ignore` instead of `ignore` here to not override the defaults?

`W503` for instance is off by default unless defaults are overridden by `ignore` configuration.